server/test/Helpers.cc

```diff
@@ -689,17 +689,6 @@ string makeItemOutput(const ItemInfo &itemInfo)
 	    static_cast<int>(itemInfo.valueType),
 	    itemInfo.unit.c_str());
 
-	// Item category
```

Comment:
Why did remove these lines? We should assertion about categoryNames in test cases.